bot/__main__.py

```diff
@@ -15,25 +15,25 @@
 )
 
 
-async def sessionify(client):
-    client.ses = ClientSession()
-
-
 class TheBot(Bot):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.loop.create_task(sessionify(self))
+        self.loop.create_task(self.sessionify())
+
+    async def sessionify(self):
+        self.ses = ClientSession()
 
     async def close(self):
         await self.ses.close()
         await super().close()
 
 
+bot = TheBot(
```

Comment:
Personally, I would have left this initialization where it was, inside the _if_ clause, because in case the condition evaluates to false, this is technically an unused instance of TheBot.

Reply:
this way it is accessible from bot/tests/cogs.py